.evergreen/run-orchestration.sh

```diff
@@ -119,13 +119,15 @@ else
 fi
 echo "ORCHESTRATION_FILE=$ORCHESTRATION_FILE"
 
+# Copy the orchestration file so we can override it.
+cp $ORCHESTRATION_FILE $MONGO_ORCHESTRATION_HOME/config.json
```

Comment:
```suggestion
cp -f "$ORCHESTRATION_FILE" "$MONGO_ORCHESTRATION_HOME/config.json"
```
Should this command have an `-f` flag (overwrite existing `config.json`)?